VideoWeb/VideoWeb/ClientApp/src/app/app.component.ts

```diff
@@ -70,9 +76,23 @@ export class AppComponent implements OnInit, OnDestroy {
     private postConfigSetup() {
         this.checkAuth().subscribe({
             next: async (loggedIn: boolean) => {
+                console.log(`Post config setup loggged in status: ${loggedIn}`);
                 await this.postAuthSetup(loggedIn);
             }
         });
+
+        this.eventService
+            .registerForEvents()
+            .pipe(
+                tap(value => {
+                    console.log('EventReceived with value from app', value);
+                }),
+                filter(notification => notification.type === EventTypes.NewAuthorizationResult)
+            )
+            .subscribe(async (value: OidcClientNotification<AuthorizationResult>) => {
+                console.log('EventReceived with value from app', value);
```

Comment:
replace with a more meaningful message and use the logger instead of console.log